### PR TITLE
Create `@kampus/std` package

### PR DESCRIPTION
**Pull request by arabacibahadir**
Set of ts/js utilities/functions that can we use in every other workspace.

### CODE REVIEW

**Comment by Ketcap** [comment]
@arabacibahadir thanks for the PR i think we need to issue for this to see why and what needs to be done for this. If the task is not there please open it and check with @usirin with the details. 

Quick review: naming as "husky" does not feel like this is utilities but this is a husky package https://github.com/typicode/husky

**Comment by usirin** [comment]
I've asked from @arabacibahadir to do this in this exact way with this exact package name 😅

Our bad, we should've opened an issue. We talked through this on my twitch stream.

**Comment by Ketcap** [comment]
> I've asked from @arabacibahadir to do this in this exact way with this exact package name 😅
> 
> Our bad, we should've opened an issue. We talked through this on my twitch stream.


No worries let's open an issue to put needs

**Comment by usirin** [comment]
@Ketcap you're right, using husky name can cause confusions. let's go with a boring, easy to understand package name: `@kampus/std`. Standard library for kampus projects.

File: packages/std/README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# `@kampus/husky`
+# `@kampus/std`
 
 Set of utilities that can we use in every other workspace.
```
Comment (usirin) [inline packages/std/README.md]:
```suggestion
Standard library for kampus projects: group of utilities that can we use in every other workspace.
```

**Review by usirin — APPROVED** [review]
💯

**Pull request merged**